installer/windows/variables.wxi

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Include xmlns="http://schemas.microsoft.com/wix/2006/wi">
+
+    <?define Name="DB Browser for SQLite" ?>
+    <?define Team="$(var.Name) Team" ?>
+    <?define Version="3.10.99" ?>
+
+    <!--
+    ======================================================= MSI ========================================================
+
+    UpgradeCode
+        Use a different GUID to be able to install both 32-bit and 64-bit together.
+    -->
+
+    <?if $(sys.BUILDARCH)="x64"?>
+        <?define ProductName="$(var.Name) (64-bit)" ?>
+        <?define UpgradeCode="124623D9-35D6-4D2E-9474-2ADACC8BABBB" ?>
+        <?define ProgramFilesFolder="ProgramFiles64Folder" ?>
+    <?elseif $(sys.BUILDARCH)="x86"?>
+        <?define ProductName="$(var.Name) (32-bit)" ?>
+        <?define UpgradeCode="23F25B1F-9C8B-41A1-A2DC-D693B87C556B" ?>
+        <?define ProgramFilesFolder="ProgramFilesFolder" ?>
+    <?endif?>
+
+    <!--
+    =============================================== Add/Remove Programs ================================================
+    https://www.firegiant.com/wix/tutorial/com-expression-syntax-miscellanea/add-or-remove-programs-entries/
+    
+    In the Programs and Features Control Panel tool, the application can have various entries like contact information.
+    -->
+
+    <?define AppIcon="..\..\src\iconwin.ico" ?>
+    <?define AppComments="$(var.Name) is a high quality, visual, open source tool to create, design, and edit database files compatible with SQLite." ?>
+    <?define AppContact="justin@postgresql.org" ?>
+    <?define AppReadme="https://github.com/sqlitebrowser/sqlitebrowser" ?>
+    <?define AppHelpLink="https://github.com/sqlitebrowser/sqlitebrowser/wiki" ?>
+    <?define AppSupportLink="https://github.com/sqlitebrowser/sqlitebrowser/issues/new/choose" ?>
```

Comment:
Might be better to have this just go to the main issues page, instead of the `.../new/choose` one.  Having people go to the existing issues section would more easily let them search the existing issues first. :smile:

Reply:
True. I will edit it. :smiley: